src/main/java/net/snowflake/ingest/streaming/internal/DataValidationUtil.java

```diff
@@ -43,17 +39,18 @@
 
 /** Utility class for parsing and validating inputs based on Snowflake types */
 class DataValidationUtil {
+
+  private static final long SECONDS_LIMIT_FOR_EPOCH = 31536000000L;
```

Comment:
nit: can we add a comment here describing how we came up with this number?

Reply:
Comments added, this refers to constants from `SFTimestamp`